std/base64.d

```diff
@@ -1764,6 +1765,7 @@ class Base64Exception : Exception
     import std.algorithm.comparison : equal;
     import std.algorithm.sorting : sort;
     import std.conv;
+    import std.exception : assertThrown;
```

Comment:
the comment said this was for enforce...

Reply:
Oh this is in a unittest. whoops.

Reply:
Yeah, but the comment is old and wasn't enforced by the compiler.